Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/18b_CR_calib_cr_drive_amplitude.py

```diff
@@ -1,12 +1,17 @@
 """
-                                 CR_calib_cr_drive_amplitude
+        CR_calib_cr_drive_amplitude
 
 The CR_calib scripts are designed for calibrating cross-resonance (CR) gates involving a system
 with a control qubit and a target qubit. These scripts help estimate the parameters of a Hamiltonian,
 which is represented as:
     H = I ⊗ (a_X X + a_Y Y + a_Z Z) + Z ⊗ (b_I I + b_X X + b_Y Y + b_Z Z)
 
-For the calibration sequences, we employ echoed CR drive.
+The sequence extracts the six coefficients as a function of CR drive amplitude,
+corresponding to Fig. 2(b) of the referenced paper.
+
+For the calibration sequences, we use one of the following CR drive configurations:
+"direct," "direct + echo," "direct + cancel," or "direct + cancel + echo."
+
```

Comment:
Maybe we can say something about the fact that we need to choose some amplitude for the drive pulse after this node no?

Reply:
Yes, at `l.38`, we say to pick an amplitude. The parameter name is not correct and fixed it.